docs/user-guide.md

```
@@ -497,10 +504,48 @@ Provider semantics:
+Each workflow YAML phase can declare a model at `phases[].models.default`,
+which flows through `RunExecutor → base_context → JidoHarness → pi CLI
+--model` flag. The phase model overrides the provider's built-in default
+model. Example:
+
+```yaml
+phases:
+  - name: assess
+    prompt: assess.md
+    models:
+      default: claude-3-5-sonnet
+```
```

markdownlint-cli2 (0.23.2)

[warning] Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)